src/amo/components/CategoryHead/index.js

```diff
@@ -0,0 +1,99 @@
+/* @flow */
+import config from 'config';
+import invariant from 'invariant';
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import { getCanonicalURL } from 'amo/utils';
+import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME } from 'core/constants';
+import translate from 'core/i18n/translate';
+import type { AppState } from 'amo/store';
+import type { I18nType } from 'core/types/i18n';
+
+type CategoryType = {|
+  application: string,
+  description: string,
+  id: number,
+  misc: boolean,
```

Comment:
hmm..I wonder what "misc" is for?

Reply:
I see...maybe this is for the "other" category..

Reply:
that's part of the API response: https://addons-server.readthedocs.io/en/latest/topics/api/categories.html#get--api-v4-addons-categories-